pyanaconda/ui/gui/spokes/advanced_storage.py

```diff
@@ -427,9 +426,6 @@ def is_member(self, device):
 
     def setup(self, store, selected_names, disks):
         """ Set up our Z-page, but only if we're running on s390x. """
-        if not arch.is_s390():
-            return
```

Comment:
Shouldn't this be replaced by ``if not STORAGE.get_proxy(DASD).IsSupported():``?

Reply:
It doesn't have to be replaced. This page is removed from the notebook if the DASD module is not supported, so the page is never set up in that case.

Reply:
Thanks for explanation.